src/socket.rs

```diff
@@ -907,6 +907,21 @@ impl UtpSocket {
         Ok(())
     }
 
+    /// Send a keepalive packet on the stream.
+    pub fn send_keepalive(&self) {
+        if now_microseconds() - self.last_acked_timestamp > 14_000_000 {
```

Comment:
I noticed that, but you also changed behavior of `send_fast_resend_request`. Good thing the behavior was restored in commit <https://github.com/canndrew/rust-utp/commit/045d31568dafb8ceff74ff8af8355fcf3324d487>.